src/libcore/slice.rs

```diff
@@ -476,14 +475,12 @@ impl<T> SliceExt for [T] {
     }
 
     #[inline]
-    fn clone_from_slice(&mut self, src: &[T]) -> usize where T: Clone {
-        let min = cmp::min(self.len(), src.len());
-        let dst = &mut self[.. min];
-        let src = &src[.. min];
-        for i in 0..min {
-            dst[i].clone_from(&src[i]);
+    fn clone_from_slice(&mut self, src: &[T]) where T: Clone {
```

Comment:
Oh gah good eye!

(for those following along the PR [was submitted](https://github.com/rust-lang/rust/pull/31000))